### ISSUE TITLE
dnsmasq cache not disabled on UBIOS 3

### ISSUE THREAD

**Issue by jfroy** [Control-D-Inc/ctrld#111]
```
~# cat /run/dnsmasq.conf.d/dns.conf

#
# Generated automatically by ubios-udapi-server
#

# Configuration of DNS Forwarder
# Never forward plain names (without a dot or domain part)
domain-needed
# Never forward addresses in the non-routed address spaces.
bogus-priv
cache-size=10000
dns-loop-detect
...

~# cat /run/dnsmasq.conf.d/zzzctrld.conf 
# GENERATED BY ctrld - DO NOT MODIFY
no-resolv
server=127.0.0.1#5354
add-mac
add-subnet=32,128
```

The dnsmasq cache should be disabled so changes made in ControlD (such as assigning a different profile to a device) take effect promptly. NextDNS implements this by adding `max-cache-ttl=0` to its generated dnsmasq config. You could also comment-out `cache-size` in the UBIOS config file and add `cache-size=0` to the ControlD file, but this may be overwritten by UBIOS at any time.

See https://github.com/nextdns/nextdns/commit/5e0087b970af3c31c2a062a77146c59c99f38d17

**Comment by yegors** [comment]
Sounds like a reasonable suggestion. Will address in the next release after the upcoming v1.3.3 which is slated for next week (and feature frozen).

**Comment by yegors** [comment]
This is done and available in v1.3.4
